.eslintrc.json

```diff
@@ -11,6 +11,18 @@
     "react/jsx-filename-extension": [2, { "extensions": [".tsx"] }],
     "react/jsx-props-no-spreading": "off",
     "react/prop-types": "off",
-    "react/react-in-jsx-scope": "off"
+    "react/react-in-jsx-scope": "off",
+    "jsx-a11y/no-noninteractive-tabindex": "off",
+    "jsx-a11y/anchor-is-valid": "off",
+    "import/extensions": [
+      "error",
+      "ignorePackages",
+      {
+        "js": "never",
+        "jsx": "never",
+        "ts": "never",
+        "tsx": "never"
   }
+    ]
+}
```

Comment:
❓ 
can you explain to me why did you add each one of these?

Reply:
hmmm, I am still thinks that you should not disable them from here on the whole project

each one of these are there for a reason, and I think we should respect these reasons, maybe you should try to fix the code in a way that doesn't complain on this

Reply:
look up why they added this rules and what are we missing if we disable them

Reply:
and for the third one, I didn't get the idea, what will happen if you remove this?